uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/providers/IdentityProviderEndpointDocs.java

```diff
@@ -591,6 +590,7 @@ void createOAuthIdentityProvider() throws Exception {
                 fieldWithPath("config.userInfoUrl").optional(null).type(STRING).description("A URL for fetching user info attributes when queried with the obtained token authorization."),
                 fieldWithPath("config.showLinkText").optional(true).type(BOOLEAN).description("A flag controlling whether a link to this provider's login will be shown on the UAA login page"),
                 fieldWithPath("config.linkText").optional(null).type(STRING).description("Text to use for the login link to the provider"),
+                fieldWithPath("config.auth_method").optional(null).type(STRING).description("<small><mark>UAA 77.10.0</mark></small> Client authentication method. Possible strings are: client_secret_basic, client_secret_post, private_key_jwt, none."),
```

Comment:
Why does this field exist in addition to `config.authMethod` (line 607, currently).

Reply:
because it can be used in creation and update of IdP. 
Accidentally found out, that config.jwtClientAuthentication is only in create, but of course it can be used in both.
The documentation fields are not that often re-used but many duplicates are in there. Sonar does not find such duplicates - unfortunately. But therefore I have added the field several times